simple_calculator/index.html

```diff
@@ -9,10 +9,13 @@
 
     </head>
 
+    <h1 style="text-align:center">Calculator Project</h1>
     <body>
-        <h1 style="text-align:center">Calculator Project</h1>
-        <div class="cal-group">
-            <input value="0" class="output-bar"> 
+        <div class="container">
+            <div class="row">
+                <input type=button class="output-bar" value="0">
```

Comment:
일단.. type="button" 이렇게 따옴표가 들어가야 문법적으로 맞지 않을까요? 따옴표가 빠져서 지금 마우스가 손가락 모양으로 안변하는 것 같습니다.

`input`사용 하셨던 거는 `button`이나 `div`가 어떨지 싶습니다.
```suggestion
                <div class="output-bar">0</div>
```

왜냐하면, input태그는 type에 따라 file(파일업로드), button(버튼), text(텍스트박스...) 등 굉장히 많은 기능을 가지지만,
보통 뭔가를 입력받는 공간을 위해 씁니다. 개인적으로, input type="text"가 가장 일반적인 용처라고 느껴집니다.

계산기 버튼은 뭔가를 입력받는 공간(입력의 도착지점)이 아니고, 입력을 위해 이벤트를 발생시키는 공간(입력의 수단)이기 때문에 전 개인적으로 `input`이 안어울린다는 생각이 듭니다.

그리고 `div`가 뭐랄까.. 가장 특색이 없는 일반적인 녀석이랄까요? 그래서 무난합니다. 디폴트로 가지고 있는 속성들도 별로 없고요. 약간 백지같은 녀석입니다.